docs/workloads/azure/data/requirements_data_azure.md

```diff
@@ -120,6 +120,12 @@ Deployment', referring to variables required after the fundamental infrastructur
 
 </details>
 
+### Github PAT Token
+
+Within the pipelines we use an Azure DevOps task called UsePythonVersion@0 which allows us to install a specific version of Python onto the build agent. If the Python version doesn't exist on the build agent, it will download it from [Github Actions](https://github.com/actions/python-versions) however this requires a Github PAT Token otherwise you may hit by a GitHub anonymous download limit.
```

Comment:
```suggestion
Within the pipelines we use an Azure DevOps task called UsePythonVersion@0 which allows us to install a specific version of Python onto the build agent. If the Python version doesn't exist on the build agent, it will download it from [GitHub Actions](https://github.com/actions/python-versions) however this requires a GitHub PAT Token otherwise you may hit by a GitHub anonymous download limit.
```